sdk/search/Azure.Search.Documents/src/Models/RawVectorQuery.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Search.Documents.Models
+{
+    /// <summary> The query parameters for vector and hybrid search queries. </summary>
+    public partial class RawVectorQuery : VectorQuery
+    {
+        /// <summary> The vector representation of a search query. </summary>
+        public IReadOnlyList<float> Vector { get; set; }
```

Comment:
Collection properties should not be settable.

Reply:
This is a specific case for embedding where they are immutable. We've already discussed this case [here](https://github.com/Azure/azure-sdk-for-net/pull/36109#discussion_r1187853918), and @tg-msft has approved this change.

Reply:
This is a specific case for embeddings. We've already discussed this case [here](https://github.com/Azure/azure-sdk-for-net/pull/36109#discussion_r1187853918), and @tg-msft has approved this change.